core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -163,6 +163,10 @@ Hudson.AdministerPermission.Description=\
   This permission grants the ability to make system-wide configuration changes, \
   as well as perform highly sensitive operations that amounts to full local system access \
   (within the scope granted by the underlying OS.)
+Hudson.ManageJenkins.Description=\
+  This permission grants the ability to make non Permission escalating system \
+  configuration changes. \
+  System configuration which would allow the execution of arbitrary commands or code on the master requires Administer.
```

Comment:
@HCloward I think there should be some commas in that sentence somewhere but I'm not sure whre.